requirements.txt

```diff
@@ -2,3 +2,4 @@ pbr>=1.3
 fixtures
 python-subunit >= 0.0.18
 testtools >= 0.9.30
+PyYAML>=3.1.0
```

Comment:
I definitely like YAML syntax better, but maybe it would be better to use JSON or ConfigParser rather than add a new library used only here?